### ISSUE TITLE
Error building master

### ISSUE THREAD

**Issue by johnbradley** [bigdatagenomics/mango#207]
When trying to build the master branch via `mvn clean package -DskipTests` I hit this error:
```
[ERROR] /Users/.../mango/mango-core/src/main/scala/org/bdgenomics/mango/models/
FeatureMaterialization.scala:93: 
error: value getContigName is not a member of org.bdgenomics.formats.avro.Feature
[ERROR]       .mapValues(r => r.map(f => BedRowJson(Option(f.getFeatureId).
getOrElse("N/A"), Option(f.getFeatureType).getOrElse("N/A"), 
f.getContigName, f.getStart, f.getEnd)))
```
I was able to change `f.getContigName` to `f.getContig.getContigName` and it will compile, but not sure this is correct.

Details about my build environment `mvn --version`:
```
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T11:41:47-05:00)
Maven home: /usr/local/Cellar/maven/3.3.9/libexec
Java version: 1.8.0_73, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_73.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.11.5", arch: "x86_64", family: "mac"
```

Thanks,

John

**Comment by fnothaft** [comment]
Ah yes, that was caused by a [recent breaking upstream change in ADAM](https://github.com/bigdatagenomics/adam/commit/8abb47ad2447bbc0376f9b5052e2e029bdf6b604). @akmorrow13 is working to fix this in https://github.com/bigdatagenomics/mango/pull/206.

**Comment by fnothaft** [comment]
This should be resolved now by #206. I'm going to close this, but please reopen @johnbradley if you still have issues building!

**Comment by akmorrow13** [comment]
This was just fixed and merged by #206
